Frontend/autozon/src/app/core/models/user.model.ts

```diff
@@ -1,16 +1,16 @@
-import { NumberSymbol } from '@angular/common';
+import { NumberSymbol } from "@angular/common";
 
 export interface IUser{
-    id: number;
```

Comment:
Please use the change with ";", lint corrected it like that...

I'll stop at that point, please merge the dev correctly in your branch instead of deleting all the changes that happend on dev meanwhile